docs/use_cases.md

```diff
@@ -537,3 +542,29 @@ spec:
   writeOutputsToSecret:
     name: helloworld-outputs
 ```
+
+## Force unlock Terraform state
```

Comment:
@Nalum I added some docs for the force-unlock feature. Could you take a look to see if this aligns with what you implemented? Thanks!